src/pip/_internal/req/__init__.py

```diff
@@ -1,9 +1,20 @@
 import collections
 import logging
-from typing import Iterator, List, Optional, Sequence, Tuple
-
-from pip._internal.utils.logging import indent_log
-
+from functools import partial
+from typing import (
+    Any,
+    Callable,
+    Iterable,
+    Iterator,
+    List,
+    Optional,
+    Sequence,
+    Tuple,
+    Union,
+)
+
+from ..._internal.utils.logging import indent_log
+from ..utils.parallel import map_multiprocess_ordered
```

Comment:
Use absolute import instead. Relative import to parent modules break too easily on refactoring.

Reply:
:+1: